Tests/Shared.Specs/Shared.Specs.projitems

```diff
@@ -9,81 +9,79 @@
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\EnumAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\GuidAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\NullableBooleanAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\NullableEnumAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\NullableGuidAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\NullableSimpleTimeSpanAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\ObjectAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\ReferenceTypeAssertionsSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\SimpleTimeSpanAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\StringAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\TimeSpanConversionExtensionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\TimeSpanExtensions.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\TimeSpanFormatterSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Specialized\AggregateExceptionAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Specialized\AssemblyAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Specialized\DelegateAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Specialized\ExecutionTimeAssertionsSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Specialized\TaskAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Specialized\TaskOfTAssertionSpecs.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)FakeClock.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ThrowAssertionsSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TimeSpanConversionExtensionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TimeSpanExtensions.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TimeSpanFormatterSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TypeAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TypeExtensionsSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TypeSelectorSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)XAttributeAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)XAttributeFormatterSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)XDocumentAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)XDocumentFormatterSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)XElementAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)XElementFormatterSpecs.cs" />
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' != 'netcoreapp1.1' ">
```

Comment:
The failing build is probably due to these files being conditionally included before.